src/frontend/qt_sdl/LAN_Capture.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <fstream>
+
+#include "../types.h"
+
+// Header for PCAP file
```

Comment:
this doesn't need to be part of the public interface, it can go into LAN_Capture.cpp